src/SFA.DAS.Commitments.Database/Tables/AccountLegalEntities.sql

```diff
@@ -0,0 +1,13 @@
+﻿CREATE TABLE [dbo].[AccountLegalEntities]
+(
+    [Id] BIGINT NOT NULL,
+    [PublicHashedId] CHAR(6) NOT NULL, 
+    [AccountId] BIGINT NOT NULL, 
+    [Name] NVARCHAR(100) NOT NULL, 
+    [Created] DATETIME2 NOT NULL,
+    [Updated] DATETIME2 NULL,
+    [Deleted] DATETIME2 NULL,
+    CONSTRAINT [PK_AccountLegalEntities] PRIMARY KEY CLUSTERED ([Id] ASC),
+    CONSTRAINT [FK_AccountLegalEntities_Accounts_AccountId] FOREIGN KEY ([AccountId]) REFERENCES [Accounts] ([Id]),
+    INDEX [IX_AccountLegalEntities_AccountId] NONCLUSTERED ([AccountId] ASC)
+)
```

Comment:
It might be more performant if the clustered index was on account id, since the account legal entities will be frequently read for an account in one go (due to the queries in the Account entity).

Reply:
Would like to defer this decision until we see how the tables are going to be accessed within our system